sdk/scheduler/src/main/java/com/mesosphere/sdk/config/validate/ConfigValidationError.java

```diff
@@ -36,7 +40,18 @@ public static ConfigValidationError valueError(
       String configField, String configValue, String message)
   {
     // Set oldValue to null
-    return new ConfigValidationError(configField, null, configValue, message);
+    return new ConfigValidationError(configField, null, configValue, message, false);
```

Comment:
```suggestion
    return valueError(configField, configValue, message, false);
```
and also remove the `// Set oldValue to null` comment